guides/Git Shopify.md

```diff
@@ -0,0 +1,50 @@
+# Git strategy
+
+Our quick guide to working collaboratively with Git, GitHub and Shopify projects.
+
+## Create a Feature
+
+Check if `staging` branch present in repo.
+
+```
+git checkout master
+git pull
+git checkout staging
+```
+
+If it is present, work the same as always, but make Pull Requests on `staging`
+
+If not, then create and push it.
+
+```
+git checkout master
+git pull
+git checkout -b staging
+git push origin staging
+```
+
+## Production hotfix
+
+If you need to make a hotfix on production, create a new, local branch off `master`.
+
+```
+git checkout master
+git pull
+git checkout -b ${branch_name} # (* - check fusnote)
+```
+
+Then create a Pull Request on `master`.
+
+## Merge staging with master
+
+When staging branch is ready to go live, merge it with master
```

Comment:
Once the featured is complete and all Pull Requests to staging have been merged, rebase `staging` on top of `master`, give it a final review and once your are happy, merge it in.